Multiplication.cs

```diff
@@ -1,18 +1,11 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
 namespace multiplication
 {
     class Multiplication
     {
         static void Main(string[] args)
         {
-            // This program will give you the multiplication table of any number you enter
-
-            Console.WriteLine("Please enter a number");
+            Console.WriteLine("Please enter the number you want to know the multiplication table for him");
```

Comment:
Him? It is not a person.

`Please enter a number to print its multiplication table`